### PR TITLE
Redesign multitasking FAB - Closes #2683 🌶

### LINKED ISSUE #2683: Redesign FAB – Multitasking (number of tabs)
Old:

New:

Spec:

Asset: [Tab.svg](https://cl.ly/253L0q2k2o47/Tab.svg)

### PR DESCRIPTION
**Pull request by Nistenf**
This PR closes issue #2683.
screenshot_1528242781

The border in the mock image looks slightly thicker than in my screenshot though. I used the asset provided with the changes mentioned in the issue, let me know if I should change it.

### CODE REVIEW

File: app/src/main/res/drawable/tab_number_border.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (pocmo) [inline app/src/main/res/drawable/tab_number_border.xml]: nit: Please add the license header here:

```XML
<?xml version="1.0" encoding="utf-8"?>
<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
```

**Comment by pocmo** [comment]
@brampitoyo Please review screenshot :)

File: app/src/main/res/drawable/tab_number_border.xml (hunk 1 above)
Comment (Nistenf) [reply app/src/main/res/drawable/tab_number_border.xml]: Done!

**Comment by Nistenf** [comment]
New screenshot after changes by @ekager !
screenshot_1528397870
That's better

File: app/src/main/res/drawable/tab_number_border.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24.0"
+    android:viewportHeight="24.0">
+  <path
+      android:strokeWidth="1"
```
[the remark this answers is not in the document]
Comment (Nistenf) [reply app/src/main/res/drawable/tab_number_border.xml]: I tried 2 and some other inbetween values, here's an undoubtedly amateur comparison:
final2
It seems to me that the values most similar to the mock are either 1 or 1.1

**Pull request merged**